prusti-specs/src/specifications/common.rs

```diff
@@ -315,6 +315,24 @@ impl<EID, ET, AT> LoopSpecification<EID, ET, AT> {
     }
 }
 
+/// Specification of a struct.
+#[derive(Debug, Clone)]
+pub struct StructSpecification<EID, ET, AT> {
+    pub invariant: Vec<(ET, Assertion<EID, ET, AT>)>
+}
+
+impl<EID, ET, AT> StructSpecification<EID, ET, AT> {
+    pub fn new(invariant: Vec<(ET, Assertion<EID, ET, AT>)>) -> Self {
+        Self { invariant }
+    }
+    pub fn empty() -> Self {
+        Self::new(Vec::new())
```

Comment:
```suggestion
        Self::new(vec![])
```

Also is this function used?